packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -1,12 +1,22 @@
 const { Command } = require('@oclif/command');
 
-const { startSpinner, endSpinner } = require('../utils/display');
+const { startSpinner, endSpinner, formatStyles } = require('../utils/display');
 
 const inquirer = require('inquirer');
 
+const DATA_FORMATS = ['json', 'raw'];
+
 class ZapierBaseCommand extends Command {
   run() {
     this._parseFlags();
+    if (this.flags.debug) {
+      this.debug.enabled = true; // enables this.debug on the command
```

Comment:
I wonder where we initialized `this.debug`. Does oclif initialize it?

Reply:
yep, that's inherited from the base command: https://github.com/oclif/command/blob/70f67914d0a8a265a563b84c832e01c9f79b7014/src/command.ts#L74